endpoint.go

```diff
@@ -590,6 +590,12 @@ func (ep *endpoint) rename(name string) error {
 		return fmt.Errorf("watch null for network %q", n.Name())
 	}
 
+	_, ok := ep.getSandbox()
```

Comment:
@mavenugo a part from this extra check I did not touch the rename logic. There was a huge difference between the 2 and maybe is not worth the risk, WDYT?